crates/routes/src/images.rs

```diff
@@ -120,7 +121,7 @@ fn adapt_request(
       if INVALID_HEADERS.contains(key) {
         client_req
       } else {
-        client_req.header(key, value)
+        client_req.header(key.as_str(), value.as_bytes())
```

Comment:
Need to remember to change these things back once actix-web upgrades to http 1.0